src/main/java/com/cinema/servlets/MainServlet.java

```diff
@@ -0,0 +1,23 @@
+package com.cinema.servlets;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+
+public class MainServlet extends HttpServlet {
+    @Override
+    protected void doGet(
+            HttpServletRequest request,
+            HttpServletResponse response) throws ServletException, IOException {
+        response.setContentType("text/html");
+        PrintWriter out = response.getWriter();
+        out.println("<h1>" + new SimpleDateFormat("yyyy/MM/dd HH:mm:ss").format(new Date()) + "</h1>");
+        out.println("<p>Hello World!</p>");
+    }
+}
```

Comment:
1)your custom Servlet must add current Timestamp to header, but not what you have done
2)Date class is an archaism, use java.time package classes

Reply:
ok, I added this line  response.addDateHeader("Date", Instant.now().toEpochMilli()); and removed old